web/src/pages/Commit/Commit.tsx

```diff
@@ -64,26 +64,7 @@ const CommitPage = () => {
                         </div>
                     </div>
 
-                    {commit.branch !== branchName && (
```

Comment:
why did we remove this?

Reply:
Now it reads commit object and gets branch from commit, so it works on any branch now